Control coverage/Feature/VirtualMachine.md

```diff
@@ -1380,18 +1355,25 @@ Open remote management ports expose a VM/compute node to a high level of risk fr
 
 	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/View_Definition.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>) 
 
-	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/Deploy_To_Azure.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>) 
+	 [![Link to Azure Policy](https://raw.githubusercontent.com/MSFT-Chirag/AzTS-docs/main/Assets/Deploy_To_Azure.jpg)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/<policy-raw-link>)  -->
 
 ### Azure Policy or ARM API used for evaluation 
 
-- Example ARM API to list service and its related property at specified level: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceName/service/{serviceName}/tenant/access? 
+- ARM API to list Virtual Machines at subscription level: /subscriptions/{0}/providers/Microsoft.Compute/virtualMachines?api-version=2019-07-01 <br />
+**Properties:** properties.networkProfile.networkInterfaces[*].id
  <br />
-**Properties:** example-property
+
+- ARM API to list Network Interfaces at subscription level: /subscriptions/{subscriptionId}/providers /Microsoft.Network/networkInterfaces?api-version=2019-04-01 <br />
+**Properties:** networkSecurityGroup.id
  <br />
 
-- Example-2 ARM API to list service and its related property at specified level: /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceName/service/{serviceName}/tenant/access? 
+- ARM API to list Virtual Networks (and associated subnets) at subscription level: /subscriptions/{subscriptionId}/providers/Microsoft.Network/virtualNetworks
+?api-version=2019-11-01 <br />
+**Properties:** properties.subnets[*].networkSecurityGroup.id
  <br />
-**Properties:** example-property
+
+- ARM API to list Network Security Groups at subscription level: /subscriptions/{subscriptionId}/providers/Microsoft.Network/networkSecurityGroups?api-version=2019-04-01 <br />
+**Properties:** destinationPortRange, destinationPortRanges
  <br />
 
 <br />
```

Comment:
Sense Agent is an externally scanned Control and the details of the evaluation are largely unknown. Please check if this is to be really added here. 

Please check the other externally scanned Controls too, These may not be required to be listed here.

Reply:
I discussed with Tarun. He recommended to keep the controls.
Logic: for external custom if logic is in place, it will be evaluated properly

Reply:
Removed controls